chaincode/fabcar/java/src/main/java/org/hyperledger/fabric/samples/fabcar/FabCar.java

```diff
@@ -140,21 +140,21 @@ public Car createCar(final Context ctx, final String key, final String make, fin
      * @return array of Cars found on the ledger
      */
     @Transaction()
-    public Car[] queryAllCars(final Context ctx) {
+    public CarQueryResult[] queryAllCars(final Context ctx) {
         ChaincodeStub stub = ctx.getStub();
 
         final String startKey = "CAR0";
         final String endKey = "CAR999";
-        List<Car> cars = new ArrayList<Car>();
-
+        List<CarQueryResult> queryResults = new ArrayList<CarQueryResult>();
+        
```

Comment:
Please, remove whitespaces.

Reply:
Fixed [in](https://github.com/hyperledger/fabric-samples/pull/149/commits/1ada9a7ca354c9a3dcfe23e1e209ac333ac54dba)